tests/test_server.py

```diff
@@ -316,6 +318,25 @@ def version(self):
 
     print(MockServer(remote_server).version)
     with pytest.raises(
-        ValueError, match="Error connecting to DPF LegacyGrpcServer with version 1.0"
+            ValueError, match="Could not connect to DPF legacy gRPC server with version 1.0."
```

Comment:
This needs to be reverted as this is only what we expect from the server. Please change it server side if it needs changing.